cobigen/cobigen-openapiplugin-parent/cobigen-openapiplugin/src/main/java/com/devonfw/cobigen/openapiplugin/inputreader/OpenAPIInputReader.java

```diff
@@ -411,13 +412,15 @@ private PropertyDef extractReferenceProperty(OpenApi3 openApi, Schema componentS
             try {
                 if (Overlay.isReference(param, "schema")) {
```

Comment:
Could you maybe add _"schema"_ as a constant? If you have a look at the [Constant](https://github.com/MikeSchumacherCapgemini/tools-cobigen/blob/fca94a2e5e1e00e48fcb5e50049de36d7bc81ecf/cobigen/cobigen-openapiplugin-parent/cobigen-openapiplugin/src/main/java/com/devonfw/cobigen/openapiplugin/util/constants/Constants.java) file, maybe you could add it there.

Therefore the final result would be:

`if (Overlay.isReference(param, Constants.SCHEMA)) {`